elastic/integrations
# [elastic/integrations] Rabbitmq_1015: Add shovel datastream

## What does this PR do?



This PR adds a new data-stream shovel for the Rabbitmq package. Shovel status  indicates if messages are being picked up and dropped off from the queue. Checking this plugin is included in the [RabbitMQ monitoring guide](https://www.rabbitmq.com/monitoring.html#readiness-probes).

## Checklist


## Author's Checklist



## How to test this PR locally



Setup the RabbitMQ(if using docker image, prefer: rabbitmq:3-management) and enable the shovel plugin(rabbitmq_shovel and rabbitmq_shovel_management)
Create a shovel to test in Rabbitmq (Admin -> Shovel Management -> Add a new shovel)
Add rabbitmq integration in UI and data should flow successfully in Kibana.

## Related issues




- Relates #https://github.com/elastic/integrations/issues/1015

## Screenshots
